docs/index.md

```diff
@@ -113,17 +115,18 @@
 ## 7. Matchups
 
 1. Roster Format
-   1. Each Team has a roster of players. Unless otherwise stated in these Bylaws, each Team's roster may never include more than 14 players.
-      1. For the 2020 season, each Team's roster may include one additional player.
-         1. This player must have an "Out" or "Injured Reserve" injury designation. When this player loses the required injury designation, the Team's roster must be adjusted to remove a player unless another player on the roster carries the required injury designation.
-         1. This player may not be added to the Lineup of their Team.
+   1. Each Team has a roster of players. Unless otherwise stated in these Bylaws, each Team's roster may never include more than 15 players.
+      1. Up to one of these players may be designated as the IR Slot. If no player is designated as the IR Slot then the team's roster may include no more than 14 players.
+         1. This player must have an "Out" or "Injured Reserve" injury designation at the time they are designated as the IR Slot and at the time of any subsequent alterations of the roster.
+         1. If the player designated as the IR Slot later loses the required injury designation, the Team's roster may not be altered until a different rostered player is legally designated as the IR Slot or no player is designated as the IR Slot.
```

Comment:
As far as I can tell, once a player gets healthy the website will let you keep him in the IR slot as long as you want so long as you don't try to drop/add anyone or put him into the starting lineup, so I tried to make the description here match that behavior.